packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -15,21 +15,32 @@ export const ResponsiveGridRoot = styled(Grid.Root)`
   container-type: inline-size;
 `;
 
-export const ResponsiveGridItem = styled(Grid.Item)`
-  grid-column: span 12;
-
-  @container (min-width: ${RESPONSIVE_CONTAINER_BREAKPOINTS.sm}) {
-    ${({ col }) => col && `grid-column: span ${col};`}
-  }
-`;
+// We need to use a different grid item for the responsive layout in the test environment
+// because @container is not supported in jsdom and it throws an error
+export const ResponsiveGridItem =
+  process.env.NODE_ENV !== 'test'
+    ? styled(Grid.Item)<{ col: number }>`
+        grid-column: span 12;
+        @container (min-width: ${RESPONSIVE_CONTAINER_BREAKPOINTS.sm}) {
+          ${({ col }) => col && `grid-column: span ${col};`}
+        }
+      `
+    : styled(Grid.Item)<{ col: number }>`
+        grid-column: span 12;
+      `;
```

Comment:
I'm really not a fan of this, can you at least put a TODO to fix it.

Reply:
I can remove this fix and add a todo to find a solution to remove warnings when we launch the FE unit tests, is it ok for you?

Reply:
Wait are they warnings or errors? If it doesn't make the test fail then I'd say remove this